Core/GDCore/Events/Builtin/LinkEvent.cpp

```diff
@@ -44,6 +44,7 @@ const EventsList * LinkEvent::GetLinkedEvents(const gd::Project & project) const
 
 void LinkEvent::ReplaceLinkByLinkedEvents(gd::Project & project, EventsList & eventList, std::size_t indexOfTheEventInThisList)
 {
+    linkWasInvalid = false;

```

Comment:
Ok very true! Maybe a comment should be added after these lines, or at the end of the function to tell that the event was potentially just destroyed?